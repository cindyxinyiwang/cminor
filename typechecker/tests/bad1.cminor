// declare twice

a: function integer () = {
	i: integer = 1;
	i: boolean = true;
}