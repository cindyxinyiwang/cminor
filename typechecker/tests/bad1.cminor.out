resolve error: i is already declared
